src/main/java/org/folio/service/caches/MappingParametersCache.java

```diff
@@ -27,7 +27,8 @@
 @Component
 public class MappingParametersCache {
   private static final Logger LOGGER = LogManager.getLogger();
-  public static final String ORGANIZATIONS = "/organizations/organizations";
+  public static final String ORGANIZATIONS = "/organizations/organizations?limit=%s";
+  public static final int ORGANIZATIONS_LIMIT = 1000;
```

Comment:
let's make it configurable, something like in SRM 
https://github.com/folio-org/mod-source-record-manager/blob/master/mod-source-record-manager-server/src/main/java/org/folio/services/mappers/processor/MappingParametersProvider.java#L96

Reply:
Thanks, done